compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/SkikoUIView.uikit.kt

```diff
@@ -165,8 +165,13 @@ internal class SkikoUIView(
     /**
      * https://developer.apple.com/documentation/uikit/uiview/1622533-point
      */
-    override fun pointInside(point: CValue<CGPoint>, withEvent: UIEvent?): Boolean =
-        delegate.pointInside(point, withEvent)
+    override fun pointInside(point: CValue<CGPoint>, withEvent: UIEvent?): Boolean {
+        val (frameWidth, frameHeight) = frame.useContents { size.width to size.height }
+        val (x, y) = point.useContents { x to y }
+        return x in 0.0..frameWidth &&
+            y in 0.0..frameHeight &&
```

Comment:
Can we use `Rect.contains` function here?

Reply:
We can't. Because Rect has offset. And we need to check point in Rect(0, 0, frameWidth, frameHeight)

Reply:
```suggestion
        val frameRect = frame.useContents { Rect(0, 0, size.width, size.height) }
        val pointOffset = point.useContents { Offset(x, y) }
        return frameRect.contrains(pointOffset) &&
```